### PR TITLE
RF 1.0.2.1 Documentation - Contributors Guide.md

### PR DESCRIPTION
**Pull request by EtricKombat-zz**
### To raise a new Pull Request, the following prerequisites need to be met. Please tick before proceeding:

- [x] a conversation was held in the appropriate media [hyperledgerkochi](https://github.com/orgs/hyperledgerkochi/projects/3) channel.
- [x] a Github [Issue](https://github.com/hyperledgerkochi/KOBman/issues) was opened for this feature / bug.
- [ ] test coverage was added .

**This addresses Issue** https://github.com/hyperledgerkochi/KOBman/issues/25

Along with the contributors guide , the below listed changes has been incorporated in this pull request

- docs/How to contribute to this project.md file removed 
-  minor edits in ISSUE_TEMPLATE.md template
-  minor edits in process-to-raise-a-pull-request.md , process-to-raise-an-issue.md

### CODE REVIEW

**Comment by EtricKombat-zz** [comment]
@asa1997  , requesting LGTM for https://github.com/asa1997/KOBman/pull/36

File: docs/user-guides/contributors_guide.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Contributing to KOBman
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
```
Comment (asa1997) [inline docs/user-guides/contributors_guide.md]: Submitting a code fix?

File: docs/user-guides/contributors_guide.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://guides.github.com/introduction/flow/index.html)). We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `master`.
```
Comment (asa1997) [inline docs/user-guides/contributors_guide.md]: @EtricKombat Can you check whether you can add the GitHub instructions for forking. Use "Fork the repo" as name of the link.

File: docs/user-guides/contributors_guide.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+## Processes expected to follow by each contributors:
+
+- [Process for pull request](https://github.com/hyperledgerkochi/KOBman/blob/master/docs/user-guides/process-to-raise-a-pull-request.md)
+
+- [Process for issues](https://github.com/hyperledgerkochi/KOBman/blob/master/docs/user-guides/process-to-raise-an-issue.md)
+
+
+
+## Use a Consistent Coding Style
+It is recommended to follow the [KOBman Checklist](https://github.com/hyperledgerkochi/KOBman/blob/master/docs/KOBman%20Script%20Review%20Checklist.md) before submitting any pull requests.
```
Comment (asa1997) [inline docs/user-guides/contributors_guide.md]: You must follow the coding standard described in the KOBman review checklist.

**Comment by asa1997** [comment]
@EtricKombat I have suggested some changed. Please go through it.

File: docs/user-guides/contributors_guide.md (hunk 3 above)
Comment (EtricKombat-zz) [reply docs/user-guides/contributors_guide.md]: @asa1997  , the wording has been updated like suggested

File: docs/user-guides/contributors_guide.md (hunk 1 above)
Comment (EtricKombat-zz) [reply docs/user-guides/contributors_guide.md]: @asa1997 removed

**Comment by asa1997** [comment]
@panickervinod @sumodgeorge LGTM

**Pull request merged**